lib/omnibus/fetchers/git_fetcher.rb

```diff
@@ -203,7 +203,7 @@ def contains_revision?(rev)
     #   the shellout object
     #
     def git(command)
-      shellout!("git #{command}", cwd: project_dir)
+      shellout!("git -c core.autocrlf=false #{command}", cwd: project_dir)
```

Comment:
I wonder if this belongs in https://github.com/chef-cookbooks/omnibus/blob/master/recipes/_git.rb rather than forcing it on every platform? I'm just worried about unintended consequences.